### PR TITLE
Add do not show the command to be executed in the window

### PR DESCRIPTION
**Pull request by kratos83**

### CODE REVIEW

**Comment by palinek** [comment]
What is the background for this? Hidding the command that is being executed with elevated privileges is a security flaw IMO.

**Comment by kratos83** [comment]
Good morning, I added this feature because both kde and gnome give this possibility.

Add background view command
lxqt_sudo_view_command

Add background not 
lxqt_sudo_not_view_command
view command

File: sudo.cpp [hunk 1]
```diff
@@ -161,6 +164,15 @@ int Sudo::main()
         {
             mBackend = BACK_SUDO;
             mArgs.removeAt(0);
+        } else if ("-r" == arg1)
+        {
+            mArgs.removeAt(0);
+            m_remove = "-r";
```
Comment (palinek) [inline sudo.cpp]: Why comparing here, assigning the very same to `m_remove` (which I see as a very confusing name for local variable) and then the same comparison in `PasswordDialog` constructor?

File: sudo.cpp [hunk 2]
```diff
@@ -46,6 +46,7 @@
 #include <fcntl.h>
 #include <iostream>
 #include <thread>
+#include <QDebug>
```
Comment (palinek) [inline sudo.cpp]: ?

**Comment by palinek** [comment]
>  I added this feature because both kde and gnome give this possibility.

Any references?

...and even if kde/gnome does provide something like that, I do not agree to do it here.

**Comment by tsujan** [comment]
> Hiding the command that is being executed with elevated privileges is a security flaw IMO.

Yes, it is.

**Comment by agaida** [comment]
another point:  normally lxqt-sudo will not called from the command line so i agree with @tsujan and palinek. Not really the behaviour we want.

**Comment by agaida** [comment]
another point:  normally lxqt-sudo will not called from the command line so i agree with @tsujan and palinek. Not really the behaviour we want.

**Pull request closed without merging**